build_tools/ci/build_test_cpp.sh

```diff
@@ -32,6 +32,9 @@ if [[ "$OSTYPE" == "linux-gnu"* ]]; then
   export CMAKE_TOOLCHAIN_FILE="$this_dir/linux_default_toolchain.cmake"
   export CC=clang
   export CXX=clang++
+else
```

Comment:
I believe for git-bash and MinGW it should be 'msys". Could test and assert on that.

Reply:
That's correct. I didn't put it because I wasn't sure we were gonna go with bash scripts for Windows users. After our convo I've decided I'm going to write the pwsh version of this same script and just use it in CI too.